src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -0,0 +1,10 @@
+package mate.academy.service;
+
+import mate.academy.exception.AuthenticationException;
+import mate.academy.model.User;
+
+public interface AuthenticationService {
+    User login(String email, String password) throws AuthenticationException;
+
+    User register(String email, String password) throws AuthenticationException;
```

Comment:
on tasks in this method do not need to be used "throws AuthenticationException;"